### PR TITLE
[T] (quests) fertile_wastes

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: Autumn秋殇 <2924297411@qq.com>
Co-authored-by: 炽焰 <119685011+DrunkFire@users.noreply.github.com>
Co-authored-by: 努力 <m18344860937@163.com>
Co-authored-by: 青丘雨 <1040214676@qq.com>
Co-authored-by: 琳琳 <604896778@qq.com>
Co-authored-by: 藏青 <2291848664@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 1]
```diff
@@ -118,11 +118,11 @@
 			]
 		}
 		{
-			title: "Boil 'em, Mash 'em"
+			title:"煮（捣）碎它们"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:121]:
```suggestion
			title:"煮熟，然后捣烂它们"
```

**Comment by Ricky-fight** [comment]
差不多没问题了

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 2]
```diff
@@ -3,24 +3,24 @@
...
-	subtitle: ["Crop farming"]
+	subtitle: ["作物种植"]
 	default_quest_shape: ""
 	default_hide_dependency_lines: false
 	quests: [
 		{
 			x: 8.0d
 			y: -4.5d
-			subtitle: "Literally watching grass grow."
+			subtitle: "让草生长出来（字面意思）"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:15]:
```suggestion
			subtitle: "确实地看着草生长"
```

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 3]
```diff
@@ -83,11 +83,11 @@
 		{
 			x: 9.0d
 			y: -4.0d
-			subtitle: "Get a Carrot"
+			subtitle: "获得胡萝卜"
 			description: [
-				"&6Golden Carrots&r seem to be surprisingly common here. In addition to a surprisingly pleasant crunch, they provide me with &65 minutes of Night Vision&r!"
+				"&6金胡萝卜&r在这里似乎出奇地普遍。除了令人惊喜的霉味，它们还为我提供了&65分钟的夜视&r！"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:88]:
```suggestion
				"&6金胡萝卜&r在这里似乎出奇地普遍。除了令人惊喜的美味，它们还为我提供了&65分钟的夜视效果&r！"
```

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 4]
```diff
@@ -83,11 +83,11 @@
...
 				""
-				"Golden Carrots can be peeled on the &6Cutting Board&r to get the perfectly preserved &bCarrot&r inside. After that, it's just a matter of farming."
+				"金胡萝卜可以在&6砧板&r上被剥皮，里面的&b胡萝卜&r完好无损。在那之后，就是种田的事了。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:90]:
```suggestion
				"金胡萝卜可以在&6砧板&r上被剥皮，里面的&b胡萝卜&r完好无损。毕竟，这很科学。"
```

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 5]
```diff
@@ -135,13 +135,13 @@
...
 			x: 9.0d
 			y: -7.0d
-			subtitle: "Make a Farm"
+			subtitle: "建造农场"
 			description: [
-				"Though these fools lack the minds for intelligent conversation, their simple nature means they don't complain when asked to tend the &bFarm&r."
+				"虽然这些笨蛋缺乏智慧对话的头脑，但他们简单的天性意味着当被要求照看&b农场&r。时，他们不会抱怨。"
 				""
-				"Without a source of water, tilling simple Dirt won't do. Tilled &6Rich Soil&r does not need water, and cannot be trampled."
+				"没有水源，耕种普通的泥土都是不行的。而耕种&6沃土&r不需要水，也不会被踩踏。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:142]:
```suggestion
				"没有水源，耕种普通的泥土都是不行的。而耕种&6沃土&r不需要水，也不会被踩坏。"
```

File: config/ftbquests/quests/chapters/fertile_wastes.snbt [hunk 6]
```diff
@@ -335,22 +335,22 @@
...
+				"所以&6召唤动物们&r应当是重中之重。"
 				""
-				"The &bStove&r is essentially a more civilized Campfire, functioning nearly identically."
+				"而&b炉灶&r本质上是一个更先进，功能更齐全的篝火。"
 				""
-				"The &bSkillet&r works heldheld or placed, so long as it's near enough to a heat source. It's 5x faster than the Campfire."
+				"还有在用&b煎锅&r时可以手持或者放下来，只要它离火源足够近。他的工作速度可以比篝火快5倍。"
 				""
-				"The &bCooking Pot&r requires a heat source, and has recipes exclusive to it. Some may require a container to serve."
+				"最后&b厨锅&r需要一个火源才能工作，并且它有自己独特的食谱。其中还有一些需要容器才能出锅。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/fertile_wastes.snbt:353]:
```suggestion
				"最后&b厨锅&r需要一个热源才能工作，并且它有自己独特的食谱。其中还有一些需要容器才能出锅。"
```

Comment (Ricky-fight) [reply config/ftbquests/quests/chapters/fertile_wastes.snbt:353]: 收回那句话。

File: config/ftbquests/quests/chapters/fertile_wastes.snbt (hunk 2 above)
Comment (xinyihl) [reply config/ftbquests/quests/chapters/fertile_wastes.snbt:15]: 这还不如不改

**Pull request merged**